### PR TITLE
Normalise comments

### PR DESCRIPTION
**Pull request by bonjourmauko**
This pull request changes several small issues regarding comments :

- Adds accents where appropriate
- Normalise new lines 
- Rename `MaReform` to `MaReforme` (as in the comments)
- Conjugate all comments to the plural second person

### CODE REVIEW

File: python/A_creer_reforme_parametrique.py [hunk 1]
```diff
@@ -1,18 +1,19 @@
 # -*- coding: utf-8 -*-
 
+# Importez OpenFisca
 import openfisca_france
-from reforme_parametrique import MaReform
+from reforme_parametre import MaReforme
```
Comment (Anna-Livia) [inline python/A_creer_reforme_parametrique.py]: `reforme_parametre`does not exist, `reforme_parametrique` does

File: python/B_calculer_avec_reforme_parametrique.py [hunk 2]
```diff
@@ -1,12 +1,11 @@
 # -*- coding: utf-8 -*-
 
+# Importez OpenFisca
 import openfisca_france
 from openfisca_core.simulations import Simulation
 from reforme_parametrique import MaReform
```
Comment (Anna-Livia) [inline python/B_calculer_avec_reforme_parametrique.py]: from reforme_parametrique import MaReforme

File: python/C_calculer_avec_reforme_structurelle.py [hunk 3]
```diff
@@ -1,29 +1,24 @@
 # -*- coding: utf-8 -*-
 
-####### Importez OpenFisca  ###########
-
+# Importez OpenFisca
 import openfisca_france
 from openfisca_core.simulations import Simulation
-from reforme_structurelle import MaReform
+from reforme_variable import MaReforme
```
Comment (Anna-Livia) [inline python/C_calculer_avec_reforme_structurelle.py]: `reforme_structurelle`

File: python/A_creer_reforme_parametrique.py (hunk 1 above)
Comment (maukoquiroga) [reply python/A_creer_reforme_parametrique.py]: Oh ok thanks!

**Review by Anna-Livia — APPROVED** [review]

**Comment by maukoquiroga** [comment]
🙂

**Pull request merged**